projects/canopy/src/lib/canopy.module.ts

```diff
@@ -24,6 +24,7 @@ import { LgPrefixModule } from './prefix/prefix.module';
 import { LgPromoCardModule } from './promo-card/promo-card.module';
 import { LgQuickActionModule } from './quick-action/quick-action.module';
 import { LgSeparatorModule } from './separator/separator.module';
+import { LgSideNavModule } from './side-nav/side-nav.module';
```

Comment:
There is a bit of an open question about names, but we can potentially ignore that for now.